.gitignore

```diff
@@ -1,6 +1,5 @@
 .*
 !./.travis.yml
-!./.gitignore
```

Comment:
I don't think we need to modify this line, you can use `git add --force` to add the newly added `.gitignore` file.
Or, we can just remove this line.